_data/toc.yaml

```diff
@@ -1,5 +1,5 @@
 toc:
-- title: Welcome to the Docs
+- title: What is Docker?
```

Comment:
Some possibilities: 

Docker Documentation
Documentation Home
(No entry at all) 

I think these are all better than a left-nav entry that says "What is Docker?" 

cc @sanscontext @londoncalling @mstanleyjones 

WDYT?